app/src/main/java/pl/kossa/akainotes/fragments/notes/NotesFragment.kt

```diff
@@ -1,26 +1,31 @@
 package pl.kossa.akainotes.fragments.notes
 
 import android.os.Bundle
-import android.util.Log
 import android.view.View
 import android.widget.Toast
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
 import androidx.recyclerview.widget.ItemTouchHelper
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
-import com.google.android.material.snackbar.Snackbar
 import kotlinx.android.synthetic.main.fragment_notes.*
 import pl.kossa.akainotes.R
 import pl.kossa.akainotes.SwipeDeleteCallback
 import pl.kossa.akainotes.adapters.NotesRvAdapter
 import pl.kossa.akainotes.data.Note
 import pl.kossa.akainotes.extensions.makeCancelSnackbar
+import pl.kossa.akainotes.fragments.viewNote.NoteViewViewModel
 
 class NotesFragment : Fragment(R.layout.fragment_notes) {
 
+    private val viewModel = NoteViewViewModel
```

Comment:
Jeszcze ten dziwny ViewModel przerób na przekazywanie tytułu i opisu przez NavArgs. Zaraz wrzucę ten kod z koła żebyś mógł się posiłkować

Reply:
To powinno być w tym commicie ale sie nie dodało, już poprawiam